Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/SqlServer/IntegrationTests/SqlProviderExecuteScalarQueryTest.cs

```diff
@@ -66,7 +66,7 @@ public void CollectionQuery ()
           () => Provider.ExecuteScalarQuery(QueryFactory.CreateQuery(Queries.GetMandatory("OrderQuery"))),
           Throws.ArgumentException
               .With.ArgumentExceptionMessageEqualTo(
-                  "Expected query type is 'Scalar', but was 'Collection'.", "query"));
+                  "Query type 'CollectionReadOnly' did not match with any of the expected query types 'ScalarReadOnly', 'ScalarReadWrite'.", "query"));
```

Comment:
For some reason, we test the ArgumentException from RdbmsProvider here in this test. It's possible that it is not easy to test directly with RdbmsProvider or that the test got refactored to here by accident.